src/debugger/appWorker.ts

```diff
@@ -60,9 +60,9 @@ export class SandboxedAppWorker {
 
     private static PROCESS_MESSAGE_INSIDE_SANDBOX = "onmessage({ data: postMessageArgument });";
 
-    constructor(sourcesStoragePath: string, debugAdapterPort: number, postReplyToApp: (message: any) => void, {
+    constructor(private packagerPort: number, sourcesStoragePath: string, debugAdapterPort: number, postReplyToApp: (message: any) => void, {
```

Comment:
the `private` keyword is used to implicitly generate a private member, correct? I'm not a big fan of this syntax, I think instance variables should be declared explicitly for better readability